src/User.php

```diff
@@ -7221,25 +7221,25 @@ public function validatePassword(string $password, array &$errors = []): bool
         // Validate security policies
         if ($CFG_GLPI["use_password_security"]) {
             if (Toolbox::strlen($password) < $CFG_GLPI['password_min_length']) {
-                $errors[] = __('Password too short!');
+                $errors[] = __s('Password too short!');
             }
             if ($CFG_GLPI["password_need_number"] && !preg_match("/[0-9]+/", $password)) {
-                $errors[] = __('Password must include at least a digit!');
+                $errors[] = __s('Password must include at least a digit!');
             }
             if ($CFG_GLPI["password_need_letter"] && !preg_match("/[a-z]+/", $password)) {
-                $errors[] = __('Password must include at least a lowercase letter!');
+                $errors[] = __s('Password must include at least a lowercase letter!');
             }
             if ($CFG_GLPI["password_need_caps"] && !preg_match("/[A-Z]+/", $password)) {
-                $errors[] = __('Password must include at least a uppercase letter!');
+                $errors[] = __s('Password must include at least a uppercase letter!');
             }
             if ($CFG_GLPI["password_need_symbol"] && !preg_match("/\W+/", $password)) {
-                $errors[] = __('Password must include at least a symbol!');
+                $errors[] = __s('Password must include at least a symbol!');
             }
         }
 
         // Validate password history
         if (!PasswordHistory::getInstance()->validatePassword($this, $password)) {
-            $errors[] = __('Password was used too recently.');
+            $errors[] = __s('Password was used too recently.');
         }
```

Comment:
These error messages should not be encoded here. Indeed, this method is used in multiple contexts, and it is preferable to encode these error in the caller context, when it is necessary.

Reply:
Fixed in latest commit.